src/REPLMode/command_declarations.jl

```diff
@@ -362,21 +362,27 @@ Redoes the changes from the latest [`undo`](@ref).
 [   :name => "add",
     :api => Registry.add,
     :should_splat => false,
-    :arg_count => 1 => Inf,
+    :arg_count => 0 => Inf,
```

Comment:
```suggestion
    :arg_count => (0 => Inf),
```
For improved clarity.

Reply:
It's the same to me but changing that in one place out of 16 looks more like inconsistency than clarity.

Reply:
Fair enough—I was just looking at the review code, not the code around it.